README.md

```diff
@@ -457,3 +457,13 @@ test$.subscribe((result) => {
 // Logs the following:
 // Got number: 5
 ```
+
+## Publishing the package
+
+The package is published manually right now.
+
+Steps to publish:
+
+1. Bump the version in src/package.json
```

Comment:
This step you can use `npm version (patch|minor|major)` to automatically increase by one depending on the change 👍

Reply:
Oh nice, I didn't know this existed. It won't work for us (there are two `package.json` files that need updating, we don't want an auto commit with just the version bump etc.) but it's cool to know it's possible